pilot/pkg/serviceregistry/serviceentry/controller_test.go

```diff
@@ -943,7 +943,7 @@ func TestWorkloadInstanceFullPush(t *testing.T) {
 		Name:      "additional-name",
 		Namespace: selectorDNS.Name,
 		Endpoint: &model.IstioEndpoint{
-			Address:        "4.4.4.4",
+			Addresses:      []string{"4.4.4.4"},
```

Comment:
we need some tests with multiple IPs

Reply:
Hi @howardjohn, sure, I add some tests for it, the change list as below:

1. modified:   pilot/pkg/serviceregistry/serviceentry/controller_test.go
2. modified:   pilot/pkg/serviceregistry/serviceentry/conversion_test.go
3. modified:   pilot/pkg/serviceregistry/serviceentry/store_test.go
All changes are for adding multiple addresses.